packages/redux/src/actions.ts

```diff
@@ -42,7 +41,9 @@ export const setWorkspaceStorage: PayloadActionCreator<WorkspaceStorage> = works
   payload: workspaceStorage,
 });
 
-export const setCommandHistoryStorage: PayloadActionCreator<CommandHistoryStorage> = commandHistoryStorage => ({
+export const setCommandHistoryStorage: PayloadActionCreator<
+  Record<string, unknown>
+> = commandHistoryStorage => ({
```

Comment:
Why is this changing in this PR?

Reply:
I guess console was creating a circular reference in the tsconfig, so can't be imported.